crates/env_filter/src/parser.rs

```diff
@@ -1,25 +1,60 @@
 use log::LevelFilter;
+use std::error::Error;
+use std::fmt::{Debug, Display, Formatter};
 
 use crate::Directive;
 use crate::FilterOp;
 
+/// Error during logger directive parsing process.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct ParsingError {
```

Comment:
nit: Feel like this should be named `ParseError` rather than `ParsingError`

Reply:
Done